lms/templates/seq_module.html

```diff
@@ -23,6 +23,13 @@
     </button>
     <nav class="sequence-list-wrapper" aria-label="${_('Sequence')}">
       <ol id="sequence-list" role="tablist">
+        % if gated_content['gated']:
+        <li>
+          <button class="active nav-item tab" aria-hidden="true" disabled>
```

Comment:
Since we have a meaningful icon here, it needs a textual equivalent.  You can either add screen reader text to the button, or add a ```title="Content Locked"``` to the button element.  A third option is to add an ```aria-label``` to the button, but I favor the title attribute here since it will also be available to mouse users who hover over the icon.    It may seem like a screen reader user wouldn't be able to access a "disabled" button, but they can.  They just won't be able to operate it, and its disabled state will be conveyed to them.  This is why it needs some sort of text value.

More importantly, this button doesn't seem to have all of the other attributes that other tabs in this tablist should have e.g. 
```
<button class="seq_video inactive nav-item tab" role="tab" tabindex="-1" aria-selected="false" aria-expanded="false" aria-controls="seq_content">[...]</button>
``` 

This control should emulate in functionality all of the behavior of our existing tab panel widget. I would look at what we have in production for that.

Reply:
fixed